src/constants/numbers.ts

```diff
@@ -0,0 +1 @@
+export const NOT_ASSIGNED_ORDERS_BLOCK = -1;
```

Comment:
why this in general constans, not in the local

Reply:
done